src/translations/es.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Another benefit of this is that we can pass them through AI (like ChatGPT) really quickly to translate. Is it perfect? No, but it's pretty damn good.